### ISSUE TITLE
[BITV] Add title to AppSidebar component

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#35858]
## Problem

To show a tooltip on a AppSidebar we currently use v-tooltip. An example can be found in https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcAppSidebar/NcAppSidebar.vue#L198, https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcAppSidebar/NcAppSidebar.vue#L234, https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcAppSidebar/NcAppSidebar.vue#L242. AppSidebar will be used in several places incl. files app.

v-tooltip is not accessible and we want to migrate to native tooltips (title attribute) instead. An overview can be found in https://github.com/nextcloud/nextcloud-vue/issues/2503.

## Solution

Add a `title` prop to the headings inside of AppSidebar component that will render as `title` attribute on the HTML element.
